test/unit/accord/utils/Property.java

```diff
@@ -555,7 +555,7 @@ default void checkPostconditions(State state, Result expected,
         default void process(State state, SystemUnderTest sut) throws Throwable
         {
             checkPostconditions(state, apply(state),
-                    sut, run(sut));
+                                sut, run(sut));
```

Comment:
nit: Might as well one-liner this one if we're going to reformat

Reply:
this is a personal style choice.  `state, apply(state)` is everything going on for the `state`, and `sut, run(sut)` is the same for `sut`... i felt this was easier to read as 2 lines rather than 1.

If you do not agree, i can normalize by making one line; just giving my argument